.gitignore

```diff
@@ -40,3 +40,6 @@ Thumbs.db
 
 # Env
 .env*.local
+.history/.env_20220119141622
```

Comment:
We need to add the `.history` folder all together as there are files that do not belong to the repo